app/views/katello/api/v2/repositories/base.json.rabl

```diff
@@ -25,6 +25,16 @@ node :content_label do |repo|
   repo.content.try(:label)
 end
 
+child :published_in_versions => :versions do |version|
+    attributes :id, :version
+    node :cv_id do |object|
```

Comment:
would you be OK with expanding these field names ie content_view_id instead of cv_id?